src/mbgl/renderer/tile_pyramid.cpp

```diff
@@ -53,7 +53,7 @@ void TilePyramid::finishRender(PaintParameters& parameters) {
 }
 
 std::vector<std::reference_wrapper<RenderTile>> TilePyramid::getRenderTiles() {
-    return { renderTiles.begin(), renderTiles.end() };
+    return sortedTiles;
```

Comment:
Instead of maintaining two vectors in parallel, can we just sort `renderTiles`, and create the reference-wrapped copy of the vector in this function?

Reply:
`RenderTile` does not have a default constructor, so `std::swap` cannot handle it, therefore standard algorithms look not applicable there :-/